tests/common.go

```diff
@@ -9500,3 +9510,24 @@ func GenerateS3BucketPolicy(sid string, encryptionPolicy string, bucketName stri
 
 	return policy, nil
 }
+
+// GetSseS3EncryptionType fetches SSE type for S3 bucket from the environment variable
+func GetSseS3EncryptionType() (api.S3Config_Sse, error) {
+	var sseType api.S3Config_Sse
+	s3SseTypeEnv := os.Getenv("S3_SSE_TYPE")
+	if s3SseTypeEnv != "" {
+		sseDetails, err := s3utils.GetS3SSEDetailsFromEnv()
+		if err != nil {
+			return sseType, err
+		}
+		switch strings.ToUpper(string(sseDetails.SseType)) {
+		case string(s3utils.SseS3):
+			sseType = api.S3Config_SSE_S3
+		default:
+			return sseType, fmt.Errorf("failed to sse s3 encryption type not valid: [%v]", sseDetails.SseType)
```

Comment:
Do we need to pass `api.S3Config_Invalid` here?

Reply:
No we need error out , since the encryption type is invalid